.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
     - ./scripts/install-dependencies.rb
 
 script:
-    - ./scripts/install-robolectric.sh
+    - ./gradlew clean assemble install test --info --stacktrace --continue
```

Comment:
@xian You added that from a previous commit. I actually meant to remove in completely. The build build should fail once there is any sign of an issue. Should I remove it?